<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtlPage</class>
 <widget class="QWidget" name="AtlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>745</height>
   </rect>
  </property>
 <item row="0" column="0">
  <widget class="QLineEdit" name="searchEdit"/>
 </item>
 <item row="0" column="1">
  <widget class="QPushButton" name="searchButton">
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </item>
 <item row="0" column="0" colspan="2">
  <layout class="QGridLayout" name="gridLayout">
    <widget class="QTreeView" name="packView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTextBrowser" name="packDescription"/>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
     <item row="0" column="2">
      <widget class="QComboBox" name="versionSelectionBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="sortByBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Version selected:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>packView</tabstop>
  <tabstop>versionSelectionBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
